ticket_test.go

```diff
@@ -151,7 +151,7 @@ func TestOpenStore(t *testing.T) {
 
 func TestWriteTo(t *testing.T) {
 	t.Parallel()
-	want := `[{"ID": 1, "subject": "This is a test ticket"}]`
+	want := `[{"subject": "This is a test ticket","ID": 1}]`
```

Comment:
This is interesting. Does the JSON encoder always order the fields deterministically? If not, what decides the ordering? If it can change from test run to test run, we'll need a smarter way of verifying the output.